content/docs/04.5-devx/05-app-profile/00-create-app-profile.md

```diff
@@ -14,118 +14,38 @@ import Tooltip from "shared/components/ui/Tooltip";
- 
-
- * [Helm](/devx/registries/helm-registry#palettehelmregistry): Helm chart repositories are available for use. You can add additional Helm registries to Palette, visit the [Palette Helm Registry](/devx/registries/helm-registry#palettehelmregistry) documentation page for guidance.
-
- 
-
- * Manifest: Layers can be constructed using raw manifests to provision Kubernetes resources that are unavailable in Palette or Charts. Pack Manifests provide a pass-through mechanism in which additional Kubernetes resources can be orchestrated onto a cluster and the rest of the stack. Specific integrations may require the creation of Secrets or CustomResourceDefinition (CRDs). Manifest files can be attached to the layer to achieve this. 
-
-    <br />
-
-    <WarningBox>
-    When adding tiers to the App profile using manifest files, specify a namespace name to which the resource must belong.
-    </WarningBox>
-
- *  Database Services: Different database applications such as MongoDB, PostgreSQl, Redis, etc.
-
-    <br />
-
-    <InfoBox>
-
-    The applications follows a hierarchy, where in the output variables values from the application added first can be passed to application tiers added after that. The reverse is not possible.
-
-    To view the available output variables from the below tiers in the code editor, type in ```{{.spectro.}}```.
-    </InfoBox>
-
+4. Select one of the available services to start configuring your App Profile. Refer to [App Profiles](/devx/app-profile) for a list of available services. 
 
+5. Provide configuration information for the service you are adding.
 
-When you select a service, you will see the configuration options for that service on the righthand side of the screen. Once you have provided all the required information for the service you may click on the **Review** button to proceed. You may also add additional services or tiers, by clicking on the blue **Actions** button on the top lefthand of the screen. 
+6. You can add more services to the App Profile if needed. To do this, click the **Actions** button next to the **Configure tier** pane. To rearrange layers, select a service and drag it up or down in the pane. Each service becomes a layer in the App Profile stack in the order shown in this pane.
 
-<br />
-
-
-5. **Review** the configuration and click on the **Finish Configuration** button.
-
-
-The Palette Dev Engine App Profile is now created and and can be used for Apps deployment. To edit or delete the App Profile, navigate to the **App Profile** page. Select the desired App Profile and click on the  **Settings** button at the top righthand side. 
-
-<br />
+7. When you've provided the required configuration information for services, click **Review**. 
 
-## Validation
```

Comment:
Why are we removing the validation steps? Every how-to needs a validation.

Reply:
Validation section is back in.